language-server/modules/langserver-compiler/src/main/java/org/ballerinalang/langserver/compiler/format/FormattingNodeTree.java

```diff
@@ -2557,16 +2563,49 @@ public void formatListConstructorExprNode(JsonObject node) {
                                         .get(FormattingConstants.NEW_LINE_COUNT).getAsInt()) + indentation);
                             }
                         }
-                    } else if (text.equals(Tokens.CLOSING_BRACKET) || text.equals(Tokens.COMMA)
-                            || text.equals(Tokens.CLOSING_PARENTHESES)) {
+                    } else if (text.equals(Tokens.CLOSING_PARENTHESES)) {
```

Comment:
Can we merge `if else` blocks for `CLOSING_BRACKET` and `CLOSING_PARENTHESIS` here?

Reply:
Fix with the https://github.com/ballerina-platform/ballerina-lang/pull/20969/commits/139b8d8786160b1e990bd651bfc3356948c30de9